src/__test__/Main.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import MainStat from '../components/Main';
+import store from '../redux/store';
+
+describe('Main page component testing using snapshots', () => {
+  it('Test Main component renders', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <MainStat />
+        </Provider>,
+      );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] You have done an excellent job, this projector will look great in your portfolio! 😎 However, there are still missing some of the required tests for the activity, I would suggest adding all the required tests:

<img width="795" alt="Screen Shot 2022-10-05 at 10 59 09 PM" src="https://user-images.githubusercontent.com/83717285/194211302-b2d5a1f8-8562-4c5f-bb3a-1f89772b9857.png">

Reply:
Thank you so much for this observation.

they are done! Mock connection to store and API.